www/main/settings/components/demo.html

```diff
@@ -0,0 +1,46 @@
+<div id="slide10" class="slide">
```

Comment:
![image](https://user-images.githubusercontent.com/2982784/111626367-22ec4900-87ab-11eb-8b3d-437d7498c782.png)
The background image isn't centered